src/wwElement.vue

```diff
@@ -1,393 +1,460 @@
+                this.$emit('update:content:effect', {
+                    calendarIdPath: null,
+                    calendarLabelPath: null,
+                    calendarColorPath: null,
+                });
+        },
+        'wwEditorState.boundProps.categories'(isBind) {
+            if (!isBind)
+                this.$emit('update:content:effect', {
+                    categoryNamePath: null,
+                    categoryColorPath: null,
+                    categoryColorTextPath: null,
+                });
+        },
+        /* wwEditor:end */
+        'content.defaultView': {
+            immediate: true,
+            handler(value) {
+                this.currentView = value;
+            },
+        },
+    },
+    methods: {
+        handleEventClick(event, domEvent) {
+            this.$emit('trigger-event', {
+                name: 'event:click',
+                event: {
+                    rawEventData: event.rawEventData,
+                    event: {
+                        start: event.start,
+                        end: event.end,
+                        title: event.title,
+                        content: event.content,
+                        calendar: event.split,
+                        allDay: event.allDay,
+                    },
+                    currentView: this.currentView,
+                    domEvent,
+                },
+            });
+        },
+        handleCellClick(event) {
+            const date = 'date' in event ? event.date : event;
+            const calendar = 'split' in event ? event.split : null;
+            this.$emit('trigger-event', {
+                name: 'cell:click',
+                event: {
+                    cell: { date, calendar },
+                    currentView: this.currentView,
+                },
+            });
+        },
+
+        /**
+         * Triggers whenever the active view changes. (Day, Week, Month, etc.)
+         *
+         * TS definition:
+         *     EventReadyChanged {
+         *         view: string
+         *         startDate: Date // View start - JS native Date object.
+         *         endDate: Date, // View end - JS native Date object.
+         *         firstCellDate: Date // Month view only, in case cell is out of current month - JS native Date object.
+         *         lastCellDate: Date // Month view only, in case cell is out of current month - JS native Date object.
+         *         outOfScopeEvents: Array<Event> // Month view only, all the events that are out of the current month.
+         *         events: Array<Event> // All the events in the current view.
+         *         week: number // Week number. Only returned if view is 'week'.
+         *     }
+         *
+         * @param event
+         * @see https://antoniandre.github.io/vue-cal/#ex--emitted-events
+         * @see https://github.com/antoniandre/vue-cal/issues/168#issuecomment-739544326 TS types
+         */
+        handleViewChange(props = {}) {
```

Comment:
You forgot to update the currentView local variable
`this.currentView = event.view` need to be added her, you can also rename "props" by "event" to follow your definition of @param